server/schedule/operator/operator.go

```diff
@@ -88,9 +88,12 @@ type StoreInfluence struct {
 }
 
 // ResourceSize returns delta size of leader/region by influence.
-func (s StoreInfluence) ResourceSize(kind core.ResourceKind) int64 {
+func (s StoreInfluence) ResourceSize(kind core.ResourceKind, EnableLeaderScheduleByCount bool) int64 {
```

Comment:
Do we need to change the function name? BTW, I prefer not to pass a bool value to control the inner logic.

Reply:
I think so too. I just changed it to an optional enum to avoid bool value.